### ISSUE TITLE
IBM Cloud support

### ISSUE THREAD

**Issue by teopost** [mingrammer/diagrams#226]
Add IBM cloud support

**Comment by VickyGuo0907** [comment]
@mingrammer  I just wonder what's the status of this issue now. I would like to contribute to this,  also include some OpenShift assets as well. Let me know what I could help with.

**Comment by mingrammer** [comment]
@VickyGuo0907 Yep. You can make a PR for supporting IBM cloud. I couldn't find the official icon image set for the IBM cloud yet. Could you please work on this?

**Comment by VickyGuo0907** [comment]
@mingrammer Sure! I will start working on this.  One place I could think of draw.io app, they provide plenty of IBM cloud infrastructure icons/images, it could be a start point.
